modules/building-images-using-RHEL-entitlements.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * work_with_shared_resources/using-shared-resource-csi-driver.adoc
+
+:_mod-docs-content-type: CONCEPT
+
+[id="building-images-using-RHEL-entitlements_{context}"]
+= Building images using RHEL entitlements
+
+If you have a Red Hat Enterprise Linux (RHEL) entitlement, the Insights operator automatically manages the entitlement keys using the Simple Content Access (SCA) feature. SCA allows RHEL systems to access the subscription content without manually managing the entitlement keys. The Insights operator imports your SCA entitlement keys and stores them in a secret named `etc-pki-entitlement` in the `openshift-config-managed` namespace.
+
+Previously, a cluster administrator would manually copy the `etc-pki-entitlement` secret to the required namespace. Now, in {ocp-product-title} 4.10 and later, {builds-title} can share the `etc-pki-entitlement` secret from the `openshift-config-managed` namespace with other namespaces using the Shared Resources Container Storage Interface (CSI) Driver Operator.
```

Comment:
Previously, a cluster administrator would manually copy the `etc-pki-entitlement` secret to the required namespace. Starting with {ocp-product-title} 4.10 and later, {builds-title} can share the `etc-pki-entitlement` secret from the `openshift-config-managed` namespace with other namespaces by using the Shared Resources Container Storage Interface (CSI) Driver Operator.